composer.json

```diff
@@ -35,6 +35,9 @@
         "phpspec/prophecy-phpunit": "^2.0",
         "phpunit/phpunit": "^9.3"
     },
+    "conflict": {
+        "phpspec/prophecy": "<1.12.0"
```

Comment:
Require it explicitly along with prophecy-phpunit integration for phpunit: it is not a dependency of dependency but implicit direct dependency as we use it directly via integration.